packages/web/components/EditProfileModal.tsx

```diff
@@ -302,18 +256,8 @@ export const EditProfileModal: React.FC<ProfileEditorProps> = ({
 
   const onSubmit = async (inputs: HasuraProfileProps) => {
     try {
-      if (isEmpty(dirtyFields)) {
-        return onClose();
-      }
-
-      if (!ceramic.did?.authenticated) {
-        setStatus(<Text>Authenticating DID…</Text>);
-        await ceramic.did?.authenticate();
-      }
```

Comment:
Can you remove this and have it work for users without a cached access token?

Reply:
@dysbulic there is the same authentication flow in `useSaveCeramicProfile.ts`, so this code seemed redundant to me. Thats why i removed it. 

In case there was a reason for having it in both places, then please let me know. 🙏🏽